From b24139b90c90e515dd9f40e9d2444f34e22312cf Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Karel=20Ko=C4=8D=C3=AD?= <karel.koci@nic.cz>
Date: Thu, 2 Jan 2020 14:38:01 +0100
Subject: [PATCH] target/linux/generic: enable BTRFS

Turris routers use BTRFS as root filesystem. This enables is globally
for all Turris OS builds on all kernels.
---
 config/Config-kernel.in         | 1 +
 target/linux/generic/config-6.6 | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/config/Config-kernel.in b/config/Config-kernel.in
index 64c8c63466..95192ec260 100644
--- a/config/Config-kernel.in
+++ b/config/Config-kernel.in
@@ -1281,6 +1281,7 @@ if KERNEL_IP_PNP
 endif
 
 config KERNEL_BTRFS_FS
+	def_bool y
 	bool "Compile the kernel with built-in BTRFS support"
 	help
 	  Say Y here if you want to make the kernel to be able to boot off a
diff --git a/target/linux/generic/config-6.6 b/target/linux/generic/config-6.6
index c8ebee7027..09244cccec 100644
--- a/target/linux/generic/config-6.6
+++ b/target/linux/generic/config-6.6
@@ -728,7 +728,7 @@ CONFIG_BROKEN_ON_SMP=y
 # CONFIG_BT is not set
 # CONFIG_BTRFS_ASSERT is not set
 # CONFIG_BTRFS_DEBUG is not set
-# CONFIG_BTRFS_FS is not set
+CONFIG_BTRFS_FS=y
 # CONFIG_BTRFS_FS_CHECK_INTEGRITY is not set
 # CONFIG_BTRFS_FS_POSIX_ACL is not set
 # CONFIG_BTRFS_FS_REF_VERIFY is not set
-- 
2.48.1

